lib/utils.js

```diff
@@ -52,4 +54,17 @@ async function endAllSimulatorDaemons () {
   log.debug('Finishing ending all simulator daemons');
 }
 
-export { killAllSimulators, endAllSimulatorDaemons };
+async function simExists (udid) {
+  let devices = await getDevices();
+
+  devices = _.pairs(devices).map((pair) => {
```

Comment:
There needs to be some commentary in here about what the data is we are mapping, etc, so that this chain makes sense.